api/model.js

```diff
@@ -13,6 +13,27 @@ const fetchTopics = (query) => {
 
 
 const fetchArticles = (query) => {
+    const {order, sort_by, topic } = query;
+    const acceptedSortBys = ['author', 'title', 'article_id', 'topic', 'created_at','votes','article_img_url'];
+    const acceptedOrders =['ASC','DESC'];
+    const acceptedTopics =['cats','mitch','paper']
```

Comment:
These accepted topics should be determined by a database query rather than hard-coded (your model for ticket 3 GET /api/topics may be helpful for this) 
This way if more topics were added this query would still work